Mac/BuildScript/build-installer.py

```diff
@@ -359,16 +359,16 @@ def library_recipes():
                   ),
           ),
           dict(
-              name="SQLite 3.42.0",
-              url="https://sqlite.org/2023/sqlite-autoconf-3420000.tar.gz",
-              checksum="0c5a92bc51cf07cae45b4a1e94653dea",
-              extra_cflags=('-Os '
-                            '-DSQLITE_ENABLE_FTS5 '
-                            '-DSQLITE_ENABLE_FTS4 '
-                            '-DSQLITE_ENABLE_FTS3_PARENTHESIS '
-                            '-DSQLITE_ENABLE_RTREE '
-                            '-DSQLITE_OMIT_AUTOINIT '
-                            '-DSQLITE_TCL=0 '
+              name="SQLite 3.43.1",
+              url="https://sqlite.org/2023/sqlite-autoconf-3430100.tar.gz",
+              checksum="77e61befe9c3298da0504f87772a24b0",
+	          extra_cflags=('-Os '
+                '-DSQLITE_ENABLE_FTS5 '
+                '-DSQLITE_ENABLE_FTS4 '
+                '-DSQLITE_ENABLE_FTS3_PARENTHESIS '
+                '-DSQLITE_ENABLE_RTREE '
+                '-DSQLITE_OMIT_AUTOINIT '
+                '-DSQLITE_TCL=0 '
```

Comment:
Please revert these style changes. _Only change line 362, 363, and 364._

See the following PRs for earlier SQLite version bumps; **your PR should look very similar to these**:

- gh-104624
- gh-102998
- gh-98639

Reply:
```suggestion
              extra_cflags=('-Os '
                            '-DSQLITE_ENABLE_FTS5 '
                            '-DSQLITE_ENABLE_FTS4 '
                            '-DSQLITE_ENABLE_FTS3_PARENTHESIS '
                            '-DSQLITE_ENABLE_RTREE '
                            '-DSQLITE_OMIT_AUTOINIT '
                            '-DSQLITE_TCL=0 '
```